### ISSUE TITLE
Assimp input file path problem

### ISSUE THREAD

**Issue by fredrik955** [assimp/assimp#768]
I have a problem importing this file: https://storage.googleapis.com/assimp_model_error/valve.obj
Here is my code:
```c++
const std::string filename = it->s_arg;
LOG("Importing file %s", filename.c_str());
scene = aiImportFile(filename.c_str(), 0);
if(!scene) {
	LOG("Error: Could not import model from file: %s. Assimp error message: %s", filename.c_str(), aiGetErrorString());
	return ERROR_IMPORT_FAILED;
}
```
If I run the program where the filename is "valve.obj" everything turns out alright.
If I instead set the filename equal to "./valve.obj", "../{folder name}/valve.obj" I get the following output:
Error: Could not import model from file: ./valve.obj. Assimp error message: File read error

**Comment by kimkulling** [comment]
Which Operation System are you using. If windows: does .\valve.obj works?

**Comment by fredrik955** [comment]
I am using Ubuntu.

**Comment by kimkulling** [comment]
OK, thanks for the info!

**Comment by smalcom** [comment]
Complex trouble: errors in file and software can not handle it. Initial file is https://www.artec3d.com/3d-models/tower-well-valve you can download it and see that "mtllib" directive has no argument, but must - material file name. As fast decision - remove mtllib at file begining. Pull-request in progress.

**Comment by kimkulling** [comment]
Closed by 47dd1eb227138fba6785b4506b2a5a7c46269a15.
